config.yaml

```diff
@@ -112,3 +112,34 @@ languages:
           url: https://github.com/go-gitea/
           weight: 60
           pre: github
+
+  pt-br:
+    weight: 3
+    languageName: Português brasileiro
```

Comment:
The better IMHO would be:

- Using "Português Brasileiro"
- And changing the footer having a <select> for languages instead to solve the space problem

@tboerger Are you OK with that?